Kernel/AspectSymfonyKernel.php

```diff
@@ -37,7 +38,7 @@ protected function configureAop(AspectContainer $container)
     public function init(array $options = [])
     {
         // it is a quick way to check if loader was enabled
-        $wasDebugEnabled = class_exists(DebugClassLoader::class, false);
+        $wasDebugEnabled = class_exists('\Symfony\Component\Debug\DebugClassLoader', false);
```

Comment:
S*** - didn't know that... I can revert it, it is not big deal either way...?

Do you want me to change goaop-symfony-bundle/GoAopBundle.php as well? Same code is used there by you, like, if you want to keep some consistency...